lib/floe/workflow/context.rb

```diff
@@ -139,7 +140,7 @@ def dig(*args)
       end
 
       def to_h
-        @context
+        @context.except("Credentials")
```

Comment:
I thikn we're also going to want to avoid printing Credentials in `.inspect` (because it's part of the `@context` ivar). That is, Context.new.inspect will print Credentials and we should hide that.  I have code I can share that will do that.

Reply:
Done

Reply:
:+1: sure thing, done